### ISSUE TITLE
add any advertisement in the recipes list

### ISSUE THREAD

**Issue by wilder** [WilderPereira/Reciclo#6]
use mopub or admob

**Comment by dougouk** [comment]
I will do this. Any ad, right? As long as I have it so that you can simply replace the ad-ID later on?

**Comment by wilder** [comment]
Yes! Any ad that can be added in the Recyclerview among its items...
Sure, I 'll replace the ad-ID later, make sure you don't commit the ad-ID string with a real value.
